aptos-move/aptos-vm/src/data_cache.rs

```diff
@@ -45,33 +43,23 @@ use std::{
 /// track of incremental changes is vital to the consistency of the data store and the system.
 pub struct StateViewCache<'a, S> {
     data_view: &'a S,
-    data_map: BTreeMap<StateKey, Option<Vec<u8>>>,
+    data_map: Cow<'a, BTreeMap<StateKey, WriteOp>>,
```

Comment:
not sure I follow the `Cow` story much but we'll talk about it

Reply:
We can get rid of the StateViewCache altogether: https://github.com/aptos-labs/aptos-core/commit/39f055092afc2b67afcd7990ba2bc6714b90682d#diff-c357acdc9a799c00c416262b0fd8e627e1f882f603e63d77625ae0f0fd9535fa